controllers/openstack_ansibleee_controller.go

```diff
@@ -274,12 +273,16 @@ func (r *OpenStackAnsibleEEReconciler) jobForOpenStackAnsibleEE(instance *redhat
 						ImagePullPolicy: "Always",
 						Image:           instance.Spec.Image,
 						Name:            instance.Spec.Name,
-						Args:            args,
 					}},
 				},
 			},
 		},
 	}
+	if len(r.AnsibleRunnerDebugCmd) != 0 {
```

Comment:
This handling is failing the CI. The length here is 1, not 0 when strings.Split is run against the empty string. I won't claim that I knew that before having to look on stackoverflow :).

https://stackoverflow.com/questions/35866221/why-does-an-empty-array-returned-from-strings-split-have-a-length-of-1-in-golang

This results in the generated pod YAML having an empty command:

https://logserver.rdoproject.org/46/146/83e55987268b8fd241a80cbb3074ac0c3d52225b/github-check/openstack-ansibleee-operator-github-rdo-integration-centos-8-crc-singlenode-centos-9-external-compute/06a291d/controller/controller/pod/deploy-external-dataplane-compute-kmmf5.yaml

And the error:

          container create failed: time="2023-04-29T02:06:55Z" level=error msg="runc create failed: unable to start container process: exec: \"\": executable file not found in $PATH"